content/js/users.md

```diff
@@ -61,6 +73,19 @@ skygear.changePassword(oldPassword, newPassword).then((user) => {
 });
 ```
 
+## Change email
+
+``` javascript
+skygear.saveUser({
+  id: '<your-user-id>', // such as skygear.currentUser.id
+  email: '<your-new-email-address>'
+}).then((user) => {
+  console.log('Email updated to:', user.email);
+}, (error) => {
+  console.error(error);
+});
+```
```

Comment:
Yes, your understanding is correct. I just afraid this will lead to expectation on changing the `user profile` too for normal user.

This is bug and fixed at 52bc11b